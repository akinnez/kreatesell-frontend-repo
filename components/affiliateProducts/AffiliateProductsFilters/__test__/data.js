export const mockData = [
  {
    id: "12",
    product_name: "Complete Facials",
    product_description: "Complete facials for woman",
    enable_preorder: "False",
    preoder_date: "",
    upload_content: "False",
    is_preoder_downloadable: "False",
    is_redirect_buyer: "False",
    product_visibility: "1",
    is_preview_only: "False",
    product_cover_picture: "",
    status: "3",
    is_active: "True",
    is_allow_affiliate: "True",
    affiliate_percentage_on_sales: "0",
    is_limited_sales: "True",
    number_of_product: "0",
    is_show_number_of_sales: "True",
    user_id: "1d3a15f6-762c-4bf0-a78f-0694e8ab7eef",
    date_created: "10/11/2021 8:25:21 PM",
    date_updated: "10/11/2021 8:47:51 PM",
    selling_price: "0.00",
    original_price: "0.00",
    product_type_id: "3",
    kreasell_product_id: "KREATE-undefined637695807219666972",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "0",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
  {
    id: "13",
    product_name: "Moulding Techniques",
    product_description: "Techniques to mould client into a narrative",
    enable_preorder: "False",
    preoder_date: "",
    upload_content: "False",
    is_preoder_downloadable: "False",
    is_redirect_buyer: "False",
    product_visibility: "0",
    is_preview_only: "False",
    product_cover_picture: "",
    status: "1",
    is_active: "true",
    is_allow_affiliate: "True",
    affiliate_percentage_on_sales: "0",
    is_limited_sales: "True",
    number_of_product: "0",
    is_show_number_of_sales: "True",
    user_id: "e1718cad-000a-41b7-99a4-af0c515aae57",
    date_created: "10/12/2021 9:38:21 AM",
    date_updated: "10/12/2021 9:38:21 AM",
    selling_price: "0.00",
    original_price: "0.00",
    product_type_id: "3",
    kreasell_product_id: "KREATE-Goks Store637696283020607552",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "0",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
  {
    id: "16",
    product_name: "Digital item3",
    product_description: "Digital Data Description yeast",
    enable_preorder: "True",
    preoder_date: "10/25/2021 5:21:06 PM",
    upload_content: "False",
    is_preoder_downloadable: "True",
    is_redirect_buyer: "False",
    product_visibility: "2",
    is_preview_only: "False",
    product_cover_picture:
      "https://res.cloudinary.com/salvoagency/image/upload/v1635014984/kreatesell/productimages/ynqyw9rcih1hydhtets0.jpg",
    status: "1",
    is_active: "True",
    is_allow_affiliate: "",
    affiliate_percentage_on_sales: "",
    is_limited_sales: "",
    number_of_product: "0",
    is_show_number_of_sales: "",
    user_id: "e1718cad-000a-41b7-99a4-af0c515aae57",
    date_created: "10/23/2021 6:49:45 PM",
    date_updated: "",
    selling_price: "0.00",
    original_price: "",
    product_type_id: "1",
    kreasell_product_id: "KREATE-moulders637706117856727450",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "1",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
  {
    id: "18",
    product_name: "Digital item 56",
    product_description: "Digital Data Description for 56",
    enable_preorder: "True",
    preoder_date: "10/25/2021 5:21:06 PM",
    upload_content: "False",
    is_preoder_downloadable: "True",
    is_redirect_buyer: "False",
    product_visibility: "2",
    is_preview_only: "False",
    product_cover_picture:
      "https://res.cloudinary.com/salvoagency/image/upload/v1635015462/kreatesell/productimages/v7do9fburhtp9jeotltm.jpg",
    status: "1",
    is_active: "True",
    is_allow_affiliate: "",
    affiliate_percentage_on_sales: "",
    is_limited_sales: "",
    number_of_product: "0",
    is_show_number_of_sales: "",
    user_id: "e1718cad-000a-41b7-99a4-af0c515aae57",
    date_created: "10/23/2021 6:57:43 PM",
    date_updated: "",
    selling_price: "0.00",
    original_price: "",
    product_type_id: "1",
    kreasell_product_id: "KREATE-moulders637706122636890342",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "1",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
  {
    id: "19",
    product_name: "Digital item 5556",
    product_description: "Digital Data Description for 56",
    enable_preorder: "True",
    preoder_date: "10/25/2021 5:21:06 PM",
    upload_content: "False",
    is_preoder_downloadable: "True",
    is_redirect_buyer: "False",
    product_visibility: "2",
    is_preview_only: "False",
    product_cover_picture:
      "https://res.cloudinary.com/salvoagency/image/upload/v1635015527/kreatesell/productimages/fyfwkdxvcfets2ie5hjc.jpg",
    status: "1",
    is_active: "true",
    is_allow_affiliate: "",
    affiliate_percentage_on_sales: "",
    is_limited_sales: "",
    number_of_product: "0",
    is_show_number_of_sales: "",
    user_id: "e1718cad-000a-41b7-99a4-af0c515aae57",
    date_created: "10/23/2021 6:58:48 PM",
    date_updated: "",
    selling_price: "0.00",
    original_price: "",
    product_type_id: "1",
    kreasell_product_id: "KREATE-moulders637706123282724739",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "1",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
  {
    id: "21",
    product_name: "Digital monogram 6",
    product_description: "Digital Data Description for 56",
    enable_preorder: "True",
    preoder_date: "10/25/2021 5:21:06 PM",
    upload_content: "False",
    is_preoder_downloadable: "True",
    is_redirect_buyer: "False",
    product_visibility: "2",
    is_preview_only: "False",
    product_cover_picture:
      "https://res.cloudinary.com/salvoagency/image/upload/v1635015628/kreatesell/productimages/ykyej6cudp3es06qqezx.jpg",
    status: "1",
    is_active: "true",
    is_allow_affiliate: "",
    affiliate_percentage_on_sales: "",
    is_limited_sales: "",
    number_of_product: "0",
    is_show_number_of_sales: "",
    user_id: "e1718cad-000a-41b7-99a4-af0c515aae57",
    date_created: "10/23/2021 7:00:30 PM",
    date_updated: "",
    selling_price: "0.00",
    original_price: "",
    product_type_id: "1",
    kreasell_product_id: "KREATE-moulders637706124305962801",
    product_listing_status: "1",
    cta_button: "BUY NOW",
    who_bears_fee: "customer",
    minimum_price: "0.00",
    is_minimum_price: "False",
    is_show_compare_price: "False",
    billing_interval_id: "1",
    pricing_type_id: "1",
    is_strike_original_price: "False",
    billing_frequency: "0",
    start_date: "",
    end_date: "",
  },
];
