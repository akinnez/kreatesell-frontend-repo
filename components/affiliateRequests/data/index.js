export const requestsData = [
  {
    id: 1,
    product_name: "UI Design Introduction",
    product_type: 2,
    kreator_name: "Jonah Jameson",
    date_created: "10/11/2021 8:25:21 PM",
    sales_price: "4568",
    currency: "NGN",
    performance: {
      sold: 50,
      visit: 123,
    },
    commission: 18,
    status: "Approved",
  },
  {
    id: 2,
    product_name: "Web Development Course",
    product_type: 3,
    kreator_name: "Mary Parker",
    date_created: "01/04/2021 8:25:21 PM",
    sales_price: "5000",
    currency: "NGN",
    performance: {
      sold: 10,
      visit: 22,
    },
    commission: 15,
    status: "Pending",
  },
  {
    id: 3,
    product_name: "Cooking Recipe",
    product_type: 1,
    kreator_name: "Peter Parker",
    date_created: "08/13/2021 8:25:21 PM",
    sales_price: "10000",
    currency: "NGN",
    performance: {
      sold: 20,
      visit: 25,
    },
    commission: 30,
    status: "Declined",
  },
  {
    id: 4,
    product_name: "My Music",
    product_type: 1,
    kreator_name: "Lil Wayne",
    date_created: "02/14/2021 8:25:21 PM",
    sales_price: "2500",
    currency: "NGN",
    performance: {
      sold: 898,
      visit: 1000,
    },
    commission: 5,
    status: "Pending",
  },
];
