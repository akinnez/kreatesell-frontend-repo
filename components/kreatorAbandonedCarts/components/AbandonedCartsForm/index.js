import { useState } from "react";
import { Typography, Button, Form, Input, Switch, Select } from "antd";
import { useFormik } from "formik";
import { MdKeyboardArrowRight, MdKeyboardArrowDown } from "react-icons/md";
import Editor from "../Editor";
import {
  emailTags,
  options,
} from "components/kreatorAbandonedCarts/data/formData";
import styles from "./index.module.scss";

const { Text } = Typography;

const AbandonedCartsForm = ({ editMailData }) => {
  const [showTags, setShowTags] = useState(false);

  const submitHandler = values => {
    console.log("submitted", values);
  };

  const formik = useFormik({
    initialValues: {
      admin_title: editMailData?.admin_title || "",
      enable: editMailData?.enable ?? false,
      when_to_send: editMailData?.when_to_send || null,
      email_subject_line: editMailData?.email_subject_line || "",
      email_content: editMailData?.email_content || "",
    },
    onSubmit: submitHandler,
  });

  const customHandler = (field, value) => {
    formik.setFieldValue(field, value);
  };

  const handleToggle = () => {
    setShowTags(!showTags);
  };

  return (
    <Form
      className={styles.form}
      name="abandoned_carts_form"
      layout="vertical"
      size="large"
      onFinish={formik.handleSubmit}
      initialValues={{
        admin_title: formik.values.admin_title,
        when_to_send: formik.values.when_to_send,
        email_subject_line: formik.values.email_subject_line,
        email_content: formik.values.email_content,
      }}
    >
      <Form.Item label="Admin Title" name="admin_title">
        <Input
          placeholder="kreatesell"
          {...formik.getFieldProps("admin_title")}
        />
      </Form.Item>
      <div className={styles.form__switch}>
        <div>
          <label htmlFor="switch">Enabled?</label>
        </div>
        <Switch
          id="switch"
          onChange={checked => customHandler("enable", checked)}
          onBlur={formik.handleBlur}
          checked={formik.values.enable}
          name="enable"
        />
        <div>
          <span>{formik.values.enable ? "Enabled" : "Disabled"}</span>
        </div>
      </div>
      <Form.Item label="When To Send" name="when_to_send">
        <Select
          placeholder="10 min"
          options={options}
          onChange={val => customHandler("when_to_send", val)}
          onBlur={formik.handleBlur}
        />
      </Form.Item>
      <Form.Item label="Email Subject Line" name="email_subject_line">
        <Input
          placeholder="kreatesell"
          {...formik.getFieldProps("email_subject_line")}
        />
      </Form.Item>
      <Editor formik={formik} />
      <div className={styles.email__tags__container}>
        <div
          className={
            showTags
              ? `${styles.email__tags__header} ${styles.email__tags__header__clicked}`
              : `${styles.email__tags__header}`
          }
        >
          <Text>Email Tags</Text>
          <Button onClick={handleToggle}>
            Show Tags&nbsp;{" "}
            {showTags ? <MdKeyboardArrowDown /> : <MdKeyboardArrowRight />}
          </Button>
        </div>
        <div
          className={
            showTags
              ? `${styles.email__tags} ${styles.email__tags__open}`
              : `${styles.email__tags}`
          }
        >
          {emailTags.map(({ id, tag, content }) => (
            <div key={id} className={styles.email__tag}>
              <Text>{`{${tag}}`}</Text>
              <Text>{content}</Text>
            </div>
          ))}
        </div>
      </div>
      <Form.Item>
        <Button type="primary" htmlType="submit">
          Save
        </Button>
      </Form.Item>
    </Form>
  );
};

export default AbandonedCartsForm;
