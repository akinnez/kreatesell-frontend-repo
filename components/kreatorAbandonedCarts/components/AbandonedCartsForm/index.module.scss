@use "public/css/colors";
@use "public/css/mixins";
@use "public/css/screens";

.form {
  div[class$="ant-form-item-label"] {
    padding-bottom: 0;

    > label {
      font-weight: 500;
      font-size: 16px;
      color: colors.$black-100;
    }
  }

  div[class^="ant-select"] > div[class="ant-select-selector"] {
    border-radius: 8px;
  }

  & > div[class$="ant-form-item"]:last-child {
    button[class^="ant-btn"] {
      height: auto;
      padding: 16px 50px;
      border-radius: 8px;
      font-weight: bold;
      font-size: 12px;
      line-height: 16px;
    }
  }
}
