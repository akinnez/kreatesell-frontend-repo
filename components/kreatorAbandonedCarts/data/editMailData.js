export const editMailData = {
  admin_title: "This is the admin title",
  enable: true,
  when_to_send: "1 week",
  email_subject_line: "This will be the subject line",
  email_content:
    '<h1 class="ql-align-center"><strong><u>Hello World</u></strong></h1><p>Dear Mr John Doe,</p><p><br></p><p>This is a test mail sample. <u>This can be underlined</u>. <strong>Look even bold works</strong> and <em>so does italics</em>.</p><p><br></p><p>Now I will make a bullet list:</p><ul><li>One</li><li>Two</li><li>Three</li></ul><p><br></p><p>And also a numbered list:</p><ol><li>One</li><li>Two</li><li>Three</li></ol><p><br></p><p>Yours Sincerely,</p><p>KreateSell.</p>',
};
