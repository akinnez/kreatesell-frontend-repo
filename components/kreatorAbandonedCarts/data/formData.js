export const emailTags = [
	{
		id: 1,
		tag: 'First_Name',
		name: 'First Name',
		content: 'Curates the first names of your email list.',
	},
	{
		id: 2,
		tag: 'Last_Name',
		name: 'Last Name',
		content: 'Curates the last names of your email list.',
	},
	{
		id: 3,
		tag: 'Product_Name',
		name: 'Product Name',
		content: 'Choose the curated names of your digital products.',
	},
	{
		id: 4,
		tag: 'Product_Price',
		name: 'Product Price',
		content: 'Shows the price of your chosen digital product.',
	},
	{
		id: 5,
		tag: 'Product_Button',
		name: 'Product Button',
		content: 'Links the recipient to the digital product page.',
	},
	{
		id: 6,
		tag: 'Checkout_Button',
		name: 'Checkout Button',
		content:
			'Links the recipient to the checkout page of the digital product.',
	},
	{
		id: 7,
		tag: 'Checkout_Url',
		name: 'Checkout Url',
		content: 'Shows the link to the checkout page for the digital product.',
	},
];

export const options = [
	{label: '10 Mins', value: '10 Mins'},
	{label: '15 Mins', value: '15 Mins'},
	{label: '30 Mins', value: '30 Mins'},
	{label: '45 Mins', value: '45 Mins'},
	{label: '1 Hour', value: '1 Hour'},
	{label: '2 Hours', value: '2 Hours'},
	{label: '4 Hours', value: '4 Hours'},
	{label: '8 Hours', value: '8 Hours'},
	{label: '12 Hours', value: '12 Hours'},
	{label: '24 Hours', value: '24 Hours'},
	{label: '36 Hours', value: '36 Hours'},
];
