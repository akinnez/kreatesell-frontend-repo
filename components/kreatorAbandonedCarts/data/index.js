export const abandonedCartsMails = [
  {
    id: 1,
    email_subject: "Please Complete Your Purchase",
    date_created: "01/13/2022 8:25:21 PM",
    time_to_send: "10 Mins",
  },

  {
    id: 3,
    email_subject: "Please Complete Your Purchase",
    date_created: "07/15/2020 8:25:21 PM",
    time_to_send: "11 Hours",
  },
  {
    id: 4,
    email_subject: "Please Complete Your Purchase",
    date_created: "02/02/2022 8:25:21 PM",
    time_to_send: "16 Days",
  },
  {
    id: 5,
    email_subject: "Please Complete Your Purchase",
    date_created: "11/31/2021 8:25:21 PM",
    time_to_send: "2 Hours",
  },
  {
    id: 6,
    email_subject: "Please Complete Your Purchase",
    date_created: "12/25/2020 8:25:21 PM",
    time_to_send: "22 Mins",
  },
  {
    id: 7,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 8,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 9,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 10,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 11,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 12,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 13,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 14,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 15,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 16,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 17,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 18,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
  {
    id: 19,
    email_subject: "Please Complete Your Purchase",
    date_created: "10/01/2021 8:25:21 PM",
    time_to_send: "25 Days",
  },
];
