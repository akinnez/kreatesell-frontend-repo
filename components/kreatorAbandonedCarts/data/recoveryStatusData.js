export const options = [
	{label: '15 Days', value: '15 Days'},
	{label: '30 Days', value: '30 Days'},
	{label: '45 Days', value: '45 Days'},
	{label: '60 Days', value: '60 Days'},
	{label: '75 Days', value: '75 Days'},
	{label: '90 Days', value: '90 Days'},
];

export const headers = [
	{label: 'Recovered By Days', key: 'recoveredByDays'},
	{label: 'Abandoned', key: 'abandoned'},
	{label: 'Recovered', key: 'recovered'},
	{label: 'In Recovery', key: 'inRecovery'},
];
