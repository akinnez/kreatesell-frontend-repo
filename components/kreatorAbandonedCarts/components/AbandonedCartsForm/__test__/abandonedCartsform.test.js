import AbandonedCartsForm from "../index";
import renderer from "react-test-renderer";
import { editMailData } from "../../../data/editMailData";

Object.defineProperty(window, "matchMedia", {
  writable: true,
  value: jest.fn().mockImplementation((query) => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(),
    removeListener: jest.fn(),
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn(),
  })),
});

it("renders a snapshot of <AbandonedCartsForm />", () => {
  const form = renderer
    .create(<AbandonedCartsForm editMailData={editMailData} />)
    .toJSON();
  expect(form).toMatchSnapshot();
});
