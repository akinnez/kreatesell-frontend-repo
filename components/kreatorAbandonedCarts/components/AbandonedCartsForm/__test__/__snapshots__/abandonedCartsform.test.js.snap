// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <AbandonedCartsForm /> 1`] = `
<form
  className="ant-form ant-form-vertical ant-form-large form"
  id="abandoned_carts_form"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <div
    className="ant-row ant-form-item"
    style={Object {}}
  >
    <div
      className="ant-col ant-form-item-label"
      style={Object {}}
    >
      <label
        className=""
        htmlFor="abandoned_carts_form_admin_title"
        title="Admin Title"
      >
        Admin Title
      </label>
    </div>
    <div
      className="ant-col ant-form-item-control"
      style={Object {}}
    >
      <div
        className="ant-form-item-control-input"
      >
        <div
          className="ant-form-item-control-input-content"
        >
          <input
            className="ant-input ant-input-lg"
            id="abandoned_carts_form_admin_title"
            name="admin_title"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="kreatesell"
            type="text"
            value="This is the admin title"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="form__switch"
  >
    <div>
      <label
        htmlFor="switch"
      >
        Enabled?
      </label>
    </div>
    <button
      aria-checked={true}
      className="ant-switch ant-switch-checked"
      id="switch"
      name="enable"
      onBlur={[Function]}
      onClick={[Function]}
      onKeyDown={[Function]}
      role="switch"
      type="button"
    >
      <div
        className="ant-switch-handle"
      />
      <span
        className="ant-switch-inner"
      />
    </button>
    <div>
      <span>
        Enabled
      </span>
    </div>
  </div>
  <div
    className="ant-row ant-form-item"
    style={Object {}}
  >
    <div
      className="ant-col ant-form-item-label"
      style={Object {}}
    >
      <label
        className=""
        htmlFor="abandoned_carts_form_when_to_send"
        title="When To Send"
      >
        When To Send
      </label>
    </div>
    <div
      className="ant-col ant-form-item-control"
      style={Object {}}
    >
      <div
        className="ant-form-item-control-input"
      >
        <div
          className="ant-form-item-control-input-content"
        >
          <div
            className="ant-select ant-select-lg ant-select-single ant-select-show-arrow"
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
          >
            <div
              className="ant-select-selector"
              onClick={[Function]}
              onMouseDown={[Function]}
            >
              <span
                className="ant-select-selection-search"
              >
                <input
                  aria-activedescendant="abandoned_carts_form_when_to_send_list_0"
                  aria-autocomplete="list"
                  aria-controls="abandoned_carts_form_when_to_send_list"
                  aria-haspopup="listbox"
                  aria-owns="abandoned_carts_form_when_to_send_list"
                  autoComplete="off"
                  className="ant-select-selection-search-input"
                  id="abandoned_carts_form_when_to_send"
                  onChange={[Function]}
                  onCompositionEnd={[Function]}
                  onCompositionStart={[Function]}
                  onKeyDown={[Function]}
                  onMouseDown={[Function]}
                  onPaste={[Function]}
                  readOnly={true}
                  role="combobox"
                  style={
                    Object {
                      "opacity": 0,
                    }
                  }
                  type="search"
                  unselectable="on"
                  value=""
                />
              </span>
              <span
                className="ant-select-selection-item"
                title="1 week"
              >
                1 week
              </span>
            </div>
            <span
              aria-hidden={true}
              className="ant-select-arrow"
              onMouseDown={[Function]}
              style={
                Object {
                  "WebkitUserSelect": "none",
                  "userSelect": "none",
                }
              }
              unselectable="on"
            >
              <span
                aria-label="down"
                className="anticon anticon-down ant-select-suffix"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="down"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                  />
                </svg>
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="ant-row ant-form-item"
    style={Object {}}
  >
    <div
      className="ant-col ant-form-item-label"
      style={Object {}}
    >
      <label
        className=""
        htmlFor="abandoned_carts_form_email_subject_line"
        title="Email Subject Line"
      >
        Email Subject Line
      </label>
    </div>
    <div
      className="ant-col ant-form-item-control"
      style={Object {}}
    >
      <div
        className="ant-form-item-control-input"
      >
        <div
          className="ant-form-item-control-input-content"
        >
          <input
            className="ant-input ant-input-lg"
            id="abandoned_carts_form_email_subject_line"
            name="email_subject_line"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="kreatesell"
            type="text"
            value="This will be the subject line"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="editorWrapper"
  >
    <div>
      <label
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        Email Content
      </label>
      <a
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Add Media
      </a>
    </div>
    <div
      className="editorContainer"
    >
      <div
        id="toolbar"
      >
        <select
          className="ql-header"
          defaultValue=""
          onChange={[Function]}
        >
          <option
            value="1"
          />
          <option
            value="2"
          />
          <option
            value="3"
          />
          <option
            value=""
          />
        </select>
        <button
          className="ql-bold"
        />
        <button
          className="ql-italic"
        />
        <button
          className="ql-underline"
        />
        <button
          className="ql-strike"
        />
        <select
          className="ql-align"
          defaultValue=""
          onChange={[Function]}
        >
          <option
            value=""
          />
          <option
            value="center"
          />
          <option
            value="right"
          />
        </select>
        <button
          className="ql-list"
          value="bullet"
        />
        <button
          className="ql-list"
          value="ordered"
        />
        <select
          className="ql-color"
          onChange={[Function]}
        />
        <button
          className="ql-link"
        />
        <button
          className="ql-image"
        />
        <button
          className="ql-video"
        />
      </div>
    </div>
  </div>
  <div
    className="email__tags__container"
  >
    <div
      className="email__tags__header"
    >
      <span
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Email Tags
      </span>
      <button
        className="ant-btn ant-btn-lg"
        onClick={[Function]}
        type="button"
      >
        <span>
          Show TagsÂ  
        </span>
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="none"
          />
          <path
            d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
          />
        </svg>
      </button>
    </div>
    <div
      className="email__tags"
    >
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {First_Name}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Curates the first names of your email list.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Last_Name}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Curates the last names of your email list.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Product_Name}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Choose the curated names of your digital products.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Product_Price}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Shows the price of your chosen digital product.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Product_Button}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Links the recipient to the digital product page.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Checkout_Button}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Links the recipient to the checkout page of the digital product.
        </span>
      </div>
      <div
        className="email__tag"
      >
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          {Checkout_Url}
        </span>
        <span
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Show the link to the checkout page for the digital product.
        </span>
      </div>
    </div>
  </div>
  <div
    className="ant-row ant-form-item"
    style={Object {}}
  >
    <div
      className="ant-col ant-form-item-control"
      style={Object {}}
    >
      <div
        className="ant-form-item-control-input"
      >
        <div
          className="ant-form-item-control-input-content"
        >
          <button
            className="ant-btn ant-btn-primary ant-btn-lg"
            onClick={[Function]}
            type="submit"
          >
            <span>
              Save
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</form>
`;
